hoot-core/src/main/cpp/hoot/core/schema/TagMergerFactory.cpp

```diff
@@ -54,7 +55,7 @@ TagMergerFactory& TagMergerFactory::getInstance()
   return *_theInstance;
 }
 
-std::shared_ptr<const TagMerger> TagMergerFactory::getDefaultPtr()
+std::shared_ptr<TagMerger> TagMergerFactory::getDefaultPtr()
 {
```

Comment:
Need to remove `const` here to allow making the `TagMerger` configurable from `TagMergerFactory`.